CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ x.x.x Release notes (yyyy-MM-dd)
 * Fix an issue where certain `LIKE` queries could hang.
 * Fix an issue where `-[RLMRealm writeCopyToURL:encryptionKey:error]` could create
   a corrupt Realm file.
+* Fix an issue where incrementing a Realm's schema version without actually
```

Comment:
From the logic of the fix, it seemed to be the sort of thing that could happen to any Realm as long as it was in the `Additive` mode. I don't think there is anything sync-related on that codepath.